flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/sort/NormalizedKeyComputer.java

```diff
@@ -29,14 +27,6 @@
  */
 public abstract class NormalizedKeyComputer {
```

Comment:
maybe we should move these interfaces which will mainly used with code generator together? like to package `org.apache.flink.table.generated` ? I created `Projection` and `JoinCondition` in previous PR and place them there.

Reply:
ok